MVC/controllers/articlesController.js

```diff
@@ -0,0 +1,10 @@
+const {selectArticleById} = require('../models/articlesModels');
+
+exports.getArticleById = (req, res, next) => {
+    const {article_id} = req.params;
+    selectArticleById(article_id)
+    .then(article => {
+        res.status(200).send({article});
```

Comment:
Here you should respond with an object that has a key of 'article' and the value is not an array, but a single object. (After refactoring the model, it should be fine.)